README.md

```diff
@@ -1114,6 +1119,117 @@ the SDK call stack. To learn more about these functions you can start by
 reading about them on [the AWS developer
 blog](https://aws.amazon.com/blogs/developer/v2-aws-sdk-for-go-adds-context-to-api-operations).
 
+### PubSub instrumentation and logging
+
+#### Kafka
+
+`go-sdk` provides a flexible way to instrument Kafka client by providing a logging, tracing and sending metrics to
+DataDog.
```

Comment:
```suggestion
`go-sdk` provides a flexible way to instrument the Kafka client with logging, tracing and metrics capabilities.
```

As far as I can see this PR does not add anything DataDog specific. It users the SDK logger and metrics capabilities which (gladly) abstract away the underlying destination.